db/db_impl.cc

```diff
@@ -2417,6 +2417,10 @@ Status DBImpl::SetOptions(ColumnFamilyHandle* column_family,
     s = cfd->SetOptions(options_map);
     if (s.ok()) {
       new_options = *cfd->GetLatestMutableCFOptions();
+      // Append new version to recompute compaction score.
+      VersionEdit dummy_edit;
```

Comment:
Did you change the impact of writing the manifest with an dummy edit? I don't know what will happen.